app/src/main/java/com/kickstarter/viewmodels/ProjectHolderViewModel.java

```diff
@@ -167,11 +177,22 @@
   }
 
   final class ViewModel extends ActivityViewModel<ProjectViewHolder> implements Inputs, Outputs {
+    private final ApolloClientType apolloClient;
     private final KSCurrency ksCurrency;
+    private final UserCurrency userCurrency;
 
     public ViewModel(final @NonNull Environment environment) {
       super(environment);
+
+      this.apolloClient = environment.apolloClient();
       this.ksCurrency = environment.ksCurrency();
+      this.userCurrency = environment.userCurrency();
+
+      this.apolloClient.userPrivacy()
+        .map(currency -> Objects.requireNonNull(currency.me()).chosenCurrency())
+        .compose(Transformers.neverError())
```

Comment:
I think this is causing the funded amount to be blank when a user is logged out or doesn't have service.
<img src=https://user-images.githubusercontent.com/1289295/47096354-0116c280-d1fd-11e8-81c0-8b24528090ea.png width=330/>

Reply:
I think adding tests for `ProjectHolderViewModel` will help you catch these edge cases!